R-packages/modeltools/R/quantgen.R

```diff
@@ -109,7 +109,7 @@
 #' @importFrom dplyr filter select pull summarize between bind_cols
 #' @importFrom tidyr pivot_longer
 #' @export
-quantgen_forecaster = function(df, forecast_date, signals, incidence_period,
+quantgen_forecaster = function(df_list, forecast_date, signals, incidence_period,
```

Comment:
This signature won't work with the new `evalcast`. Unless you pass some of these through the `forecaster_args()` list.

Reply:
Most of it is actually unnecessary though.

Reply:
The plan is to make it all compliant with the new evalcast although I realize that they are not visible from one another yet.